src/GitLabApiClient/Models/RateLimitPagedInfo.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+
+namespace GitLabApiClient.Models;
+
+public class RateLimitPagedInfo
```

Comment:
I find this not intuitive to have rate limit and page info as the same class since those are different things, I can, some API returns rate limit info regardless to page info, might want to separate to two classes (BTW those can be records but don't want to break the conventions used in this project)

Reply:
Updated